### PR TITLE
[docs] Features added to landing page and readme (closes #763)

### LINKED ISSUE #763: Docs: we need to highlight some features we forgot about
We've completely forgot to mention some important features: 

- Each TestCafe test runs in isolated client environment - tests doesn't share cookies or local and session storage values. So you don't need to manually clean up state before each test run. 

- Automatic waiting for page loads, XHRs and element appearance (selector's wait for visible). The former already highlighted in the docs, but I guess we should bring it to README and landing pages as well. 

I guess we should add it to features in README and landing page. Also we need to mention that somewhere in the docs. Also both features fits well in additional subtopic on landing page, something like "Boilerplate code free or smthg"

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
\cc @DevExpress/testcafe

### CODE REVIEW

File: .markdownlint-readme.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-    "MD009": { "br_spaces": 2 },
+    "MD002": false,
+	"MD009": { "br_spaces": 2 },
```
Comment (kirovboris) [inline .markdownlint-readme.json]: Please, fix indentation here.

File: docs/articles/index.html [hunk 2]
```diff
@@ -98,6 +97,18 @@
         </div>
         <div class="feature">
             <span class="name">
+                No Extra Coding
+            </span>
+            <div class="description">
+                <p>TestCafe keeps your tests clear of boilerplate code.<p>
```
Comment (kirovboris) [inline docs/articles/index.html]: ```<p>``` tag is not closed

**Comment by kirovboris** [comment]
lgtm

File: README.md [hunk 3]
```diff
@@ -110,7 +109,14 @@ test('Type the developer name, obtain the header text and check it', async t =>
 });
 ```
 
-## Descriptive Reports
+### No Extra Coding
+
+TestCafe keeps your tests clear of boilerplate code.
+
+* It automatically waits for page loads, XHRs and animations. Thus, there is no need to write custom code to process these tasks.
```
Comment (inikulin) [inline README.md]: Actually, we don't wait for animations

**Comment by MargaritaLoseva** [comment]
@testcafe-build-bot \retest

**Comment by MargaritaLoseva** [comment]
@testcafe-build-bot \retest

**Pull request merged**